extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -372,7 +393,39 @@ public function getCuratedContentItems( $items ) {
 	 * @param $item
 	 * @return mixed
 	 */
-	public function processCuratedContentItem( $item ) {
+	public function processCuratedContentItem( $item, $newFormat=false ) {
+		if ( $newFormat ) {
+			$result = [];
+			$result[ 'label' ] = empty($item['label']) ? $item['title'] : $item['label'];
+			$result[ 'imageUrl' ] = $item['image_url'];
+			$result[ 'type' ] = $item['type'];
+			if ( isset( $item['image_crop'] ) ) {
+				$result[ 'imageCrop' ] = $item['image_crop'];
```

Comment:
I think it'd be nice to always return this field and put null when it's not available. This way we have constant and predictable api structure